src/main/java/org/gridsuite/study/server/dto/BasicStudyInfos.java

```diff
@@ -20,10 +22,13 @@
 @Getter
 @ApiModel("Basic study attributes")
 public class BasicStudyInfos {
+    UUID id;
 
     String studyName;
 
     String userId;
 
     ZonedDateTime creationDate;
+
+    String caseFormat;
```

Comment:
why add this

Reply:
this field is required by the front app (to show the logo) 
getStudyList return now   Flux<BasicStudyInfos> instead of  Flux<StudyInfos>